site/content/docs/Upgrade/_index.md

```diff
@@ -40,22 +40,24 @@ $ kopia repository disconnect
 
 * make sure to stop any running `kopia server` instances and disable all background kopia tasks, such as periodic snapshots in `crontab`.
 
-3. Upgrade all kopia clients to the latest version >=v0.9.x
+3. Upgrade all kopia clients to the latest version >=v0.10.x
 
 4. Using the designated `kopia` client, run:
 
 ```
-$ kopia repository set-parameters --upgrade
+$ kopia repository upgrade --upgrade-owner-id <unique-id>
```

Comment:
let's discuss the CLI experience on slack, perhaps?

Reply:
UPDATE from offline talk:

We decided to maintain backwards compatibility so we do not break existing usage and also start moving to a dedicated upgrade CLI. I will also update docs around this.